src/components/Carousel/Carousel.tsx

```diff
@@ -145,16 +147,17 @@ export const Carousel: FC<CarouselProps> = ({
     >
       <ScrollContainer
         className={twMerge(theme.scrollContainer.base, (isDeviceMobile || !isDragging) && theme.scrollContainer.snap)}
-        draggingClassName="cursor-grab"
+        draggingClassName={'cursor-grab'}
         innerRef={carouselContainer}
         onEndScroll={handleDragging(false)}
-        onStartScroll={handleDragging(true)}
+        onStartScroll={handleDragging(draggable)}
         vertical={false}
+        horizontal={draggable}
       >
         {items?.map((item, index) => (
           <div
             key={index}
-            className={theme.item.wrapper}
+            className={theme.item.wrapper + (draggable ? '' : ' cursor-default')}
```

Comment:
You probably need to use `twMerge` function here. Not like the "huge" effect, but it is the default behavior for the components when handling `className` values. Also, I think that could be worth moving this style to the theme. In this case, use the `FlowbiteBoolean`, so you will have something like:

```jsx
className={twMerge(theme.item.wrapper, theme.item.wrapper.draggable[draggable ? 'on' : 'off'])}
```

And you *don't* need to do the same for the `draggingClassName`, just leave has it is. Let me ask @SutuSebastian what his opinion is as well.

Reply:
Since I only need to change the cursor-grab flag to cursor-default would it be okay solve that without twMerge but making theme.item.wrapper a Flowbiteboolean entirely.

Theme
```
item: {
    base: 'absolute top-1/2 left-1/2 block w-full -translate-x-1/2 -translate-y-1/2',
    wrapper: {
      off: 'w-full flex-shrink-0 transform cursor-default snap-center',
      on: 'w-full flex-shrink-0 transform cursor-grab snap-center',
    },
  },```

Use
```
<div
            key={index}
            className={theme.item.wrapper[draggable ? 'on' : 'off']}
            data-active={activeItem === index}
            data-testid="carousel-item"
          >
```